W-12436122-Update-WhatsNew-for-v7-15-CP (#449)
mulesoft/docs-studio: modules/ROOT/pages/whats-new-in-studio.adoc

> = What’s New in Studio

Reviewer: Are you sharing this content with the What's New section in the Release Notes? It would be good to set up a partial for this.

Author: No, the information included in this What's New page is not the same included in the RNs. This article includes more details. The version in the What's New section of the RN is shorter than this version.

Reviewer: I think they should be the same. This is not particularly long or detailed - why not include it in the RNs and single-source?

Also, we are (very soon) moving to including the Release Notes in with the docs.

> Anypoint Studio 7.15.0 improves the developer experience by enhancing Studio performance for a number of users, strengthens the asset import experience, and adds more to CloudHub deployment options.

Reviewer: ```suggestion
Anypoint Studio 7.15.0 improves the developer experience by enhancing Studio performance for Windows users, improves the asset-import experience, and adds more CloudHub deployment options.
```

Reviewer: Is this sentence necessary?

Author: It works as a summary of the new features the new version of Studio has to avoid just listing features.

== Improved Performance for Customers on Windows

> Windows users now experience improved performance in Studio, as they can use the built-in tool to exclude Studio from Windows Defender scans.

Reviewer: ```suggestion
Windows users now experience improved performance in Studio by using the built-in tool to exclude Studio from Windows Defender scans.
```

> Windows users now experience improved performance in Studio by using the built-in tool to exclude Studio from Windows Defender scans.

Reviewer: ```suggestion
Windows users now experience improved performance in Studio when they use the built-in tool to exclude Studio from Windows Defender scans.
```

== Enhanced Experience Importing API Specifications or Fragments

> Studio enhances importing experience of API specifications or fragments from Design Center improving the ability to search and provide more details displaying:

Reviewer: I'm not sure what this sentence is saying. Maybe split into two?

Author: Thanks for your comment!
I think the new version is easier to read and understand.

> Studio enhances importing experience of API specifications or fragments from Design Center. It improves the ability to search and provides more details displaying:

Reviewer: When importing API specifications or fragments from Design Center, Studio provides more information, including:

Author: Thanks!

* Type of asset
* Date the asset was last updated

== Easy Deployment and Redeployment of a Mule App to Cloudhub 2.0

> Users now can deploy and redeploy a Mule app to CloudHub 2.0 even if it has the same name and target as an existing one in CloudHub 2.0. This feature is helpful for redeploying a Mule app after making changes to it.

Reviewer: ```suggestion
You can now deploy and redeploy a Mule app to CloudHub 2.0 even if it has the same name and target as an app already deployed to CloudHub 2.0. This feature is helpful for redeploying a Mule app after making changes to it.
```

== Native Support for M1 and M2 Processors

Studio now natively supports Apple M1 and M2 processors.

== Improvements in the User Experience

> * Studio now provides more and clearer details in the log if the pom.xml has an error.

Reviewer: ```suggestion
* Studio now provides more and clearer details in the log when the `pom.xml` file contains an error.
```

> * The .mule folder is included in the .gitignore folder so displaying of changes in your project is more accurate.

Reviewer: ```suggestion
* The `.mule` folder is included in the `.gitignore` folder so displaying changes in your project is more accurate.
```
